modules/tests/test/copy_template_test.go

```diff
@@ -141,28 +99,49 @@ var _ = Describe("Copy template task", func() {
 				TaskRunTestConfig: testconfigs.TaskRunTestConfig{
 					ServiceAccount: CopyTemplateServiceAccountName,
 				},
-				TaskData: testconfigs.CopyTemplateTaskData{
-					Template:                cirrosTemplate,
-					SourceTemplateName:      cirrosTemplate.Name,
-					SourceTemplateNamespace: cirrosTemplate.Namespace,
-					TargetTemplateName:      newTemplateName,
-					TargetTemplateNamespace: f.TestNamespace,
-				},
-			}, cirrosTemplate.Namespace,
-			),
+				TaskData: testconfigs.CopyTemplateTaskData{},
+			}),
```

Comment:
it would be nice to have also tests for the namespace defaulting (when task param is `""` )